uktrade/data-hub-frontend
# Add advisers to companies list dashboard

## Description of change
This feature displays advisers in a new column of the "My companies" dashboard. 

## Test instructions
You might want to play around in your local Django for the following as there are a few scenarios to check:

1. Add a company that has an interaction with an adviser who is allocated to a team. You should see:
**Name - team**

2. Add a company that has an interaction with an adviser who is not allocated to a team. You should see:
**Name - unknown team**

3. Add a company that has an interaction with no adviser name but is associated with a team. You should see:
**Unknown adviser - team**

4. Add a company that has an interaction with multiple advisers (ie more than one). You should see:
**Multiple advisers**

5. Add a company that has an interaction with no adviser and has no team association. You should see:
**Unknown adviser - unknown team**


![Screenshot 2020-01-13 at 15 20 22](https://user-images.githubusercontent.com/10154302/72268302-7ac47880-3619-11ea-846e-037eda0fc7bb.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
